docs/linresp/relax.rst

```diff
@@ -0,0 +1,115 @@
+      5.077        0.00000000         4   ->     7        1.000       5.077      S
+      5.467        0.00126721         6   ->     8        1.000       5.457      S
+      6.000        0.00000000         3   ->     7        1.000       6.000      S
+      7.128        0.00006156         6   ->     9        1.000       7.128      S
+      7.138        0.00000000         6   ->    10        1.000       7.138      S
+      7.327        0.02861414         6   ->    11        1.000       7.224      S
+      7.446        0.00000000         5   ->     8        1.000       7.446      S
+      8.964        0.08762770         4   ->     8        1.000       8.685      S
+
+Note that we are only interested in singlet states here, since absorption to triplet states is spin forbidden. We will focus on the first excited state (S1) at 1.848 eV, which has symmetry A2. This symmetry can be inferred by inspection of the molecular orbitals involved in the transition, but we will not do so in this recipe. The state at 4.77 eV has much larger oscillator strength, but in real life also the S1 state will be very weakly absorbing due to electron-phonon coupling. This is however not taken into account at the present level of theory. Due to symmetry, the oscillator strength is exactly zero in our example.
+
+Adiabatic excitation energies
+=============================
+
+[Input: `recipes/linresp/range-separated/relax/emi`]
+
+We now relax the structure in the first excited state. The input (dftb_inp.hsd) for this task looks like this::
+
+  Geometry = GenFormat {
+      <<< "in.gen" 
+  }
+
+  Driver = LBFGS {}
+
+  Hamiltonian = DFTB {
+    
+      SCC = Yes
+      SCCTolerance = 1.0E-10
+      MaxAngularMomentum = {
+          S = "d"
+          C = "p"
+	  H = "s"
+      }
+      SlaterKosterFiles = Type2FileNames {
+          Prefix = "../../../slakos/download/mio/mio-1-1/"
+          Separator = "-"
+          Suffix = ".skf"
+      }
+  }
+
+  ExcitedState {
+      Casida {
+          NrOfExcitations = 10
+    	  StateOfInterest = 1 
+          Symmetry = Singlet
+    	  Diagonaliser = Stratmann {SubSpaceFactor = 30}
+      }
+  }
+
+  ParserOptions = {
+      ParserVersion = 10
+  }
+
+In contrast to the earlier examples in this recipe, we now set a *Driver*. We choose LBFGS optimization, but one could also take any other optimizer available in *DFTB+*. You may also set convergence criteria for the forces like detailed in the recipe *Basic usage*. Important is the new entry in the *Casida* block *StateOfInterest*. It tells the code to optimize on the S1 potential energy surface. It is recommended to choose *NrOfExcitations* always somewhat larger (i.e., +5) as *StateOfInterest*, since iterative eigensolvers might not converge to the exact solution for boundary eigenvalues. Note also that during a relexation the order of the excited states might change. This is a frequent cause of error in these kind of simulations. 
```

Comment:
```suggestion
In contrast to the earlier examples in this recipe, we now set a *Driver*. We choose LBFGS optimization, but one could also take any other optimizer available in *DFTB+*. You may also set convergence criteria for the forces like detailed in the recipe :ref:`sec-basics`. Important is the new entry in the *Casida* block *StateOfInterest*. It tells the code to optimize on the S:sub:`1` potential energy surface. It is recommended to choose *NrOfExcitations* always somewhat larger (i.e., +5) as *StateOfInterest*, since iterative eigensolvers might not converge to the exact solution for boundary eigenvalues. Note also that during a relexation the order of the excited states might change. This is a frequent cause of error in these kind of simulations. 
```